resources/fetch-gcp-projects.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Function to list projects under a folder recursively
+list_projects_recursive() {
+  local folder_id="$1"
+
+  # List projects under the current folder
+  printf " %s" $(gcloud projects list --filter="parent.id=$folder_id" --format="value(projectId)")
+
+  # List subfolders and call this function recursively
+  local subfolders=$(gcloud resource-manager folders list --folder=$folder_id --format="value(name)")
+  for subfolder in $subfolders; do
+    list_projects_recursive "$subfolder"
+  done
+}
+
+# List projects under the root organization
+org_id="933620940614"
```

Comment:
Let's remove our test orgID from the script

Reply:
done. removed it.